setup.py

```diff
@@ -172,7 +172,13 @@ def run(self):
     license='LGPLv2.1+',
     platforms=['any'],
     py_modules=['libusb1'],
-    packages=['usb1'],
+    packages=['usb1', 'usb1.__pyinstaller'],
+    include_package_data=True,
```

Comment:
This was required on my machine (Windows) to include the DLL in the package. I don't fully understand how the DLL is added in the build process, so this might not be needed.

Reply:
It is the install step which picks the dll as directed by `LIBUSB_BINARY` and hands it over to `setuptools` so it gets included in the wheel being built. In my understanding of setuptools/distutils, they do not expect to package pre-built dlls (either it is a pure package, or it needs to build something and maybe link to other libraries).

I'll check this some more.

Reply:
Having now understood that dlls are indeed package data when they are used by ctypes, and to avoid unintended distribution of files I could accidentally drop in this folder, I have replaced this line with an explicit mapping of which package data files are to be included.

Could you try with my [feature/PyInstallerHook](/vpelletier/python-libusb1/tree/feature/PyInstallerHook) branch to confirm it still works for you ?

Reply:
A quick sample of your branch worked! Also looking at the changes everything should be fine.

Reply:
Excellent, thanks.